### PR TITLE
Set tooltips for Fetch options under Pull button

### LINKED ISSUE #5330: Find a better label for "Fetch all"
**Do you want to request a *feature* or report a *bug*?**
Improvement.

**What is the expected behavior?**

As discussed [here](https://github.com/gitextensions/gitextensions/pull/5324#issuecomment-414123588), I think that the "fetch all" label is too unclear because we don't know what are "all" the things that will be fetched (branches --what most of the new git users think-- or remote repositories?).

So I think we should find a better label.

Let's discuss about that!

I propose:

1. 'fetch from all remote repositories' (which I prefer)
1. 'fetch all remote repositories' 
1. 'fetch all repositories' 
1. 'fetch from all remotes'  (but i like it less because, by metonymy, is also used to talk about branch, no?)
1. 'fetch all remotes'

### PR DESCRIPTION
**Pull request by mdonatas**
Fixes #5330

Changes proposed in this pull request:
- Add tooltips to Pull button Fetch actions
 
Screenshots after:
image

### CODE REVIEW

File: GitUI/CommandsDialogs/FormBrowse.Designer.cs [hunk 1]
```diff
@@ -527,6 +529,7 @@ private void InitializeComponent()
             this.fetchPruneAllToolStripMenuItem.Name = "fetchPruneAllToolStripMenuItem";
             this.fetchPruneAllToolStripMenuItem.Size = new System.Drawing.Size(239, 22);
             this.fetchPruneAllToolStripMenuItem.Text = "Fetch and prune all";
+            this.fetchPruneAllToolStripMenuItem.ToolTipText = "Fetch branches and tags from all remote repositories also prune deleted refs";
```
Comment (drewnoakes) [inline GitUI/CommandsDialogs/FormBrowse.Designer.cs]: Might be nice to add a bit more detail here for people who don't know what pruning is.

> Fetch branches and tags from all remote repositories, then remove (prune) any remote-tracking references that no longer exist on remotes.

Does this command also run `--prune-tags`?

Comment (mdonatas) [reply GitUI/CommandsDialogs/FormBrowse.Designer.cs]: \<sarcasm>Gosh, to get an answer to the `--prune-tags` question I had to press F12 so many times that the label on F12 key almost faded away.\</sarcasm>
And the answer is no. So how about this:
>Fetch branches and tags from all remote repositories, then remove (prune) any remote-tracking branch references that no longer exist on remotes.

Comment (mdonatas) [reply GitUI/CommandsDialogs/FormBrowse.Designer.cs]: Actually.. `--prune-tags` is not found anywhere in solution. But if we decide to add it, lets to it in another PR.

**Comment by RussKie** [comment]
All good to merge?

> Actually.. --prune-tags is not found anywhere in solution. But if we decide to add it, lets do it in another PR.

Would you be able to put a PR for this?

**Comment by mdonatas** [comment]
I think so, even though this seems low on my "want to do" list :D

File: GitUI/CommandsDialogs/FormBrowse.Designer.cs (hunk 1 above)
Comment (drewnoakes) [reply GitUI/CommandsDialogs/FormBrowse.Designer.cs]: Thanks for checking. My phone doesn't have an F12 key.

**Pull request merged**